static/img/logo.svg

```diff
@@ -1,3 +1,25 @@
+<path d="M104.502 28.7538C104.676 28.7538 104.846 28.7369 105.01 28.703C105.177 28.6692 105.343 28.6269 105.507 28.5761V29.7284C105.336 29.8029 105.123 29.8638 104.868 29.9112C104.617 29.9619 104.342 29.9873 104.042 29.9873C103.693 29.9873 103.379 29.9332 103.1 29.8249C102.824 29.7132 102.606 29.522 102.446 29.2513C102.289 28.9772 102.21 28.5964 102.21 28.1091V25.3731H101.446V24.7183L102.325 24.2005L102.786 23.0025H103.806V24.2107H105.444V25.3731H103.806V28.1091C103.806 28.3257 103.869 28.4882 103.994 28.5964C104.124 28.7013 104.293 28.7538 104.502 28.7538Z" fill="#8791AB"/>
+<path d="M108.29 24.2107V29.8858H106.694V24.2107H108.29ZM107.495 21.9873C107.732 21.9873 107.936 22.0415 108.107 22.1497C108.278 22.2546 108.363 22.4526 108.363 22.7436C108.363 23.0313 108.278 23.231 108.107 23.3426C107.936 23.4509 107.732 23.5051 107.495 23.5051C107.254 23.5051 107.048 23.4509 106.877 23.3426C106.71 23.231 106.626 23.0313 106.626 22.7436C106.626 22.4526 106.71 22.2546 106.877 22.1497C107.048 22.0415 107.254 21.9873 107.495 21.9873Z" fill="#8791AB"/>
+<path d="M115.286 27.0381C115.286 27.5118 115.219 27.9315 115.087 28.2969C114.958 28.6624 114.768 28.9721 114.517 29.2259C114.269 29.4763 113.969 29.6658 113.617 29.7944C113.268 29.923 112.874 29.9873 112.434 29.9873C112.022 29.9873 111.644 29.923 111.299 29.7944C110.957 29.6658 110.659 29.4763 110.404 29.2259C110.153 28.9721 109.957 28.6624 109.818 28.2969C109.682 27.9315 109.614 27.5118 109.614 27.0381C109.614 26.4086 109.729 25.8756 109.959 25.4391C110.189 25.0025 110.517 24.6709 110.943 24.4442C111.368 24.2174 111.876 24.1041 112.465 24.1041C113.013 24.1041 113.498 24.2174 113.92 24.4442C114.346 24.6709 114.679 25.0025 114.919 25.4391C115.164 25.8756 115.286 26.4086 115.286 27.0381ZM111.241 27.0381C111.241 27.4103 111.283 27.7233 111.367 27.9772C111.45 28.231 111.581 28.4222 111.759 28.5508C111.937 28.6794 112.169 28.7436 112.455 28.7436C112.738 28.7436 112.966 28.6794 113.14 28.5508C113.318 28.4222 113.447 28.231 113.528 27.9772C113.611 27.7233 113.653 27.4103 113.653 27.0381C113.653 26.6624 113.611 26.3511 113.528 26.1041C113.447 25.8536 113.318 25.6658 113.14 25.5406C112.963 25.4154 112.731 25.3528 112.445 25.3528C112.022 25.3528 111.716 25.4932 111.524 25.7741C111.335 26.055 111.241 26.4763 111.241 27.0381Z" fill="#8791AB"/>
+<path d="M119.927 24.1041C120.551 24.1041 121.053 24.2699 121.434 24.6015C121.814 24.9298 122.004 25.4577 122.004 26.1853V29.8858H120.408V26.5711C120.408 26.165 120.331 25.8587 120.178 25.6523C120.028 25.4458 119.791 25.3426 119.466 25.3426C118.978 25.3426 118.645 25.5034 118.467 25.8249C118.289 26.1464 118.2 26.61 118.2 27.2157V29.8858H116.604V24.2107H117.823L118.038 24.9365H118.127C118.252 24.7403 118.408 24.5812 118.592 24.4594C118.781 24.3376 118.988 24.2479 119.215 24.1904C119.445 24.1328 119.683 24.1041 119.927 24.1041Z" fill="#8791AB"/>
 </svg>
```

Comment:
The image does not change in dark theme. I think that it is unexpected behavior.

Reply:
@Bam6ycha fixed ✅
![Screenshot_177](https://user-images.githubusercontent.com/70880574/211497171-998eb389-6465-472c-9bd9-93b25c639f66.png)